plugin.video.embycon/resources/lib/downloadutils.py

```diff
@@ -346,7 +377,7 @@ def get_user_artwork(self, user, item_type):
         tag = user.get("PrimaryImageTag")
         server = self.getServer()
 
-        return "%s/emby/Users/%s/Images/%s?Format=original&tag=%s" % (server, user_id, item_type, tag)
+        return "%s/emby/Users/%s/Images/%s?Format=original&tag=%s|verifypeer=false" % (server, user_id, item_type, tag)
```

Comment:
same

Reply:
same

Reply:
use the verify_cert setting to append the verifypeer=false is needed